website/docs/configuration.md

```diff
@@ -68,7 +68,7 @@ Suppose your Alfred project has the following config:
 }
 ```
 
-This config can be extracted to an Alfred config by creating a sharable config. In this case, it makes sense to call the config `alfred-config-web-app`.
```

Comment:
Oh wow this was definitely super confusing haha. Good catch!

Reply:
Thank you!